dbt-cta/mobilize/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,804 @@
+version: 2
+
+models:
+
+  - name: affiliations_base
+    description: ''
+    columns:
+      - name: id
```

Comment:
I think it could be useful to add uniqueness tests on the primary keys of these non-ab base tables 🤷

Reply:
agreed! logically the `*_hashid` fields _should_ also function as primary keys, but it seems like a good idea to also test for uniqueness when there's another field that seems like it ought to be the obvious PK for the table (I think generally a safe assumption when it's a field called `id`). Even if those tests should be redundant, it can still be useful to have them both - and if that reveals that an "obvious" UPK is not actually unique, that might be something we document somewhere (maybe the README for the dbt project).

Bottom line, I agree with @kanelouise, let's add non-null and unique tests to all the `id` fields 😎

Reply:
makes sense! ill give it a go and try to find PKs to add tests to :catsalute: